chall01/phakakos.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import string
+import re
+
+pattern = re.compile("^[a-zA-Z ]+$")
+
+def pUsage():
+        print("usage: ./phakakos.py <a-zA-Z string>")
+        exit(1)
+
+def morse_letter(num):
+	num+=1
+	code={
```

Comment:
Instead of having numbers as keys in your dict, you could have added the letters directly. For example:
```
code = {'a': '.-', 'b': '-...'', etc}
```.